### ISSUE TITLE
[ic-toast] - aria-progressbar-name a11y violation on auto dismiss IcToast

### ISSUE THREAD

**Issue by MI6-286** [mi6/ic-ui-kit#2402]
## Summary of the bug 
Auto dismiss toast has an accessibility violation due to the loading indicator having an empty aria-label. More generally, the progress bar does not have an accessible name.

## 🪜 How to reproduce 
1. Uncomment 'cy.checkA11yWithWait()' within test the test 'it("should render auto dismiss")' of IcToast Cypress tests and run test
2. See the accessibility violation

## 📸 Screenshots or code 
![image](https://github.com/user-attachments/assets/59fde597-ff8f-4f62-a523-92403ad456c4)

## Additional info 
Issue occurs within v3.0.0/develop. On v2, the aria-label is set to 'Loading'.
